src/server/util/readingsUtils.js

```diff
@@ -184,6 +184,24 @@ const unitDatakWh = [
         displayable: Unit.displayableType.NONE,
         preferredDisplay: false,
         note: 'special unit'
+    },
```

Comment:
While I see merit in adding units & conversions used more than once to the utils file, we have tried to limit these to ones used in many/majority of tests. Given this, could you please put these units directly in the test files and with each needed test. Sorry for any confusion about this.